### ISSUE TITLE
Tests are failing in local

### ISSUE THREAD

**Issue by beatfreaker** [dalibegovic/spring-validation-custom-property-name-support#1]
I am getting below exception when running tests locally, can I get any help resolving this issue ?

```java
org.springframework.web.util.NestedServletException: Request processing failed; nested exception is java.lang.IllegalStateException: JSR-303 validated property 'personal_data.first_name' does not have a corresponding accessor for Spring data binding - check your DataBinder's configuration (bean property versus direct field access)
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1014)
	at org.springframework.web.servlet.FrameworkServlet.doPost(FrameworkServlet.java:909)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:652)
	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:883)
	at org.springframework.test.web.servlet.TestDispatcherServlet.service(TestDispatcherServlet.java:72)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
```

**Comment by aleks-shbln** [comment]
Same issue here. Wondering how it worked before and if there is a workaround to fix it now.

**Comment by dalibegovic** [comment]
Hi @beatfreaker and @aleks-shbln. This is actually the expected behaviour - sorry that I didn't write it in the readme file, I will update it.

I have created this project as an example to show what happens when [this](https://docs.jboss.org/hibernate/stable/validator/reference/en-US/html_single/#section-property-node-name-provider) feature from Hibernate Validator is used in combination with Spring.

I have opened a Spring issue [here](https://github.com/spring-projects/spring-framework/issues/24811), to see if it would make sense to support this feature in Spring. Unfortunately, no progress has been made so far.
